DOCUMENTATION.md

```diff
@@ -92,42 +92,77 @@ The `nav` attribute describes an array of navigation AsciiDoc files uses to buil
 
 The `version` attribute is the most important aspect of this file.  This designates the naming convention to be used when referencing what _version_ this documentation represents.  This can be an actual version number as shown above, in which case it should be quoted.  Other examples could be things like `stable` or `latest`.
 
+**NOTE**: As of Antora 2.3, the `antora.yml` file can now define Asciidoc attributes, which are discussed below.
+
 ### Attributes
 
-Should a documentation page need to reference attributes, e.g. version numbers, this can be accomplished by importing the `_attributes.adoc` file in the `.adoc` documentation page.  If the attribute is new, its generally a good idea to update the `_attributes.adoc` with it so that it can be reused and updated in a single place.
+Should a documentation page need to reference attributes, e.g. version numbers, this can be accomplished in two ways:
+
+1. Defined in the `antora.yml` component descriptor in this repository.
+2. Defined in the Antora playbook files in the Debezium website repository.
+
+The general rule is if an attribute changes frequently or is related to a specific version or subset of versions of Debezium, it likely belongs in the `antora.yml` file in this repository.  If the attribute changes infrequent or is not specific to a given version of Debezium, its easier to maintain that in the various playbook files in the Debezium website repository.
+
+Lets say we need to add an attribute that points to our JIRA issue for issue links, that would be an example of an attrbute that would be defined in the playbook.  But if we needed to add an attribute that points to a specific version of a naven artifact or reference a specific version of a dependency, that's more appropriate for the `antora.yml` component descriptor located in this repository. 
```

Comment:
```suggestion
Lets say we need to add an attribute that points to our JIRA issue for issue links,
that would be an example of an attrbute that would be defined in the playbook.
But if we needed to add an attribute that points to a specific version of a Maven artifact or reference a specific version of a dependency,
that's more appropriate for the `antora.yml` component descriptor located in this repository. 
```

Reply:
Fixed a typo (naven) and also changed to "one thought per line" wrapping style. Let's stick to that for easier diffing in case of future changes.